### PR TITLE
Omnibus Improvements

### PR DESCRIPTION
**Pull request by sjmiles**
Notable: remove timeout from designer-layout

### CODE REVIEW

File: pkg/Library/Mediapipe/HolisticSticker.js [hunk 1]
```diff
@@ -5,21 +5,24 @@
  * license that can be found in the LICENSE file.
  */
 ({
-async initialize(_, state, {service}) {
+async initialize({index}, state, {service}) {
```
Comment (mariakleiner) [inline pkg/Library/Mediapipe/HolisticSticker.js:8]: nit - `index` is not used in `initialize`

File: pkg/Library/NodeGraph/Editor.js [hunk 2]
```diff
@@ -1,193 +1,614 @@
...
+        {fromKey, fromStore, toKey: newNodeKey, toStore},
+        true
+      );
+      return {pipeline};
+    } else {
+      state.postAddCandidateData = undefined;
     }
-    return {selectedNode};
-  },
+    // TODO(jingjin): put the runner UI for the just-added candidate node next
```
Comment (mariakleiner) [inline pkg/Library/NodeGraph/Editor.js:76]: optional - do we want to rename the todos?

**Review by mariakleiner — APPROVED** [review]

File: pkg/Library/Mediapipe/HolisticSticker.js (hunk 1 above)
Comment (sjmiles) [reply pkg/Library/Mediapipe/HolisticSticker.js:8]: I think we will want to use it soon.

File: pkg/Library/NodeGraph/Editor.js (hunk 2 above)
Comment (sjmiles) [reply pkg/Library/NodeGraph/Editor.js:76]: Unclear, let's discuss.

**Pull request merged**